### ISSUE TITLE
`...{"": x}` not recognized as valid binding pattern

### ISSUE THREAD

**Issue by donnagf** [pvdz/zeparser3#20]
If you try this on AST Explorer you will notice it should be valid

```ts
export default {r: ("x").p = (((function* i() {
})))} = (((class extends ((this)) {
  static set [(/\xCA+/gm)] (t = x) {
  }
  static set [(((2e308)))] (s) {
  }
  set [true] ({}) {
  }
}))(((function ffky([] = ((new.target[2e308])), akpwaulpds, [p = 2e308, t, , b = ((false)), , ], ...{"": m, [null]: v = n}) {
  throw ((2e308)) <= (new.target)
  ;
  switch (false) {}
  while (((null)) in "b") for (r of new.target) continue;
}))))
```

**Comment by pvdz** [comment]
Minimal test case, I guess: `function x(...{"": m}) {}`

**Comment by pvdz** [comment]
Fixed in 3a3ae4b0d40ada7b23ae5fe664de68dd0a631040 (I think, maybe another rev)
